### PR TITLE
Update Get-MsolGroupMember.md

### PR DESCRIPTION
**Pull request by ReedK-DFT**
Incorrect example for GroupMemberType - was using text from GroupType by mistake

### CODE REVIEW

**Comment by msewaweru** [comment]
Hi @ReedK-DFT - Thanks for your contribution. Please redo this PR with the base branch as main instead of live.

**Comment by ReedK-DFT** [comment]
Changed base branch to main.

File: azureadps-1.0/MSOnline/Set-MsolCompanySettings.md [hunk 1]
```diff
@@ -38,8 +38,7 @@ This command turns on the self-serve password reset feature for all administrato
 ## PARAMETERS
 
 ### -SelfServePasswordResetEnabled
-Indicates whether to allow the use of the self-service password reset feature.
-This setting is applied company-wide.
+Indicates whether to allow the use of the self-service password reset feature for all administrators in the company.
```
Comment (msewaweru) [inline azureadps-1.0/MSOnline/Set-MsolCompanySettings.md:41]: Isn't SSPR feature available for all users, not just the administrators? Please clarify.

**Comment by CelesteDG** [comment]
Closing due to lack of activity.

File: azureadps-1.0/MSOnline/Set-MsolCompanySettings.md (hunk 1 above)
Comment (ReedK-DFT) [reply azureadps-1.0/MSOnline/Set-MsolCompanySettings.md:41]: I have no idea who made this change or who you are waiting to hear back from.  This has nothing to do with the original PR I submitted, which is needed.

**Comment by msewaweru** [comment]
> My change never got applied and is needed. I don't know why someone else's changes were merged in and I can't speak to their change, but I can guarantee that the documentation is currently wrong about GroupMemberType. Please split out my original change and apply it to the live version. The live version is still wrong!

Hi @ReedK-DFT, apologies for the mix-up. I have put your changes in a [separate PR](https://github.com/Azure/azure-docs-powershell-azuread/pull/892) as I did not receive feedback on the second change that I was not clear on. Thank you for your contribution. We'll merge that change from that PR. Thanks.

**Pull request closed without merging**